pkgs/os-specific/linux/pcm/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, callPackage, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "201710";
+  name = "pcm-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opcm";
+    repo = "pcm";
+    rev = "${version}";
+    sha256 = "02rq8739zwwbfrhagvcgf6qpmnswxl9b0qsld26rg6zp91v2npbj";
+  };
+
+  nativeBuildInputs = [ ];
+  buildInputs = [ ];
+
+  buildPhase = ''
```

Comment:
I think this is the default behavior of the build phase. So you could also remove this.